timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -89,17 +89,21 @@ TimeLockServices getOrCreateServices(String namespace) {
         return servicesByNamespace.computeIfAbsent(namespace, this::createNewClient);
     }
 
-    public int numberOfClients() {
+    public int getNumberOfActiveClients() {
         return servicesByNamespace.size();
     }
 
+    public int getMaxNumberOfClients() {
+        return maxNumberOfClients.get();
+    }
+
     private TimeLockServices createNewClient(String namespace) {
         Preconditions.checkArgument(!namespace.equals(PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE),
                 "The client name '%s' is reserved for the leader election service, and may not be "
                         + "used.",
                 PaxosTimeLockConstants.LEADER_ELECTION_NAMESPACE);
 
-        if (numberOfClients() >= maxNumberOfClients.get()) {
+        if (getNumberOfActiveClients() >= maxNumberOfClients.get()) {
```

Comment:
nit: We should standardise on using the public methods here (so we should use `getMaxNumberOfClients()`)